convention-develocity-maven-extension/README.md

```diff
@@ -1,8 +1,52 @@
 ## Convention Develocity Maven Extension
 
-This project demonstrates how a convention extension can share the same Develocity build configuration across multiple projects. It
-is intended to serve as a starting point for creating your own Maven extension that applies your specific Develocity configuration. Note
-the inline comments in the build and source code for things to adjust specifically to your needs.
+This project demonstrates how a convention extension can share the same Develocity build configuration across multiple projects.
+It is intended to serve as a starting point for creating your own Maven extension that applies your specific Develocity configuration.
+Note the inline comments in the build and source code for things to adjust specifically to your needs.
+
+### Applying the extension to your build
+
+#### Using a version range
+
+It's recommended to apply the convention extension using a [version range](https://maven.apache.org/enforcer/enforcer-rules/versionRanges.html).
```

Comment:
I wish there was a better link than the enforcer plugin, but I was not able to find one. The wiki page they linked to feels worse than this link.

Reply:
I had the exact same thought. Actually, the wiki page they link to appears to be a Maven 2.0 design document. ❓❓❓ 

![image](https://github.com/gradle/develocity-build-config-samples/assets/5797900/80f063b6-d7c3-4dad-a3d2-29a7b5b47fe7)

Reply:
lol

> last modified by Carlos Sanchez Gonzalez on Jan 24, 2006